deps/exokit-bindings/windowsystem/src/windowsystem.cc

```diff
@@ -332,8 +332,8 @@ constexpr GLint MAX_TEXTURE_SIZE = 4096;
 bool CreateRenderTarget(WebGLRenderingContext *gl, int width, int height, GLuint sharedColorTex, GLuint sharedDepthStencilTex, GLuint sharedMsColorTex, GLuint sharedMsDepthStencilTex, GLuint *pfbo, GLuint *pcolorTex, GLuint *pdepthStencilTex, GLuint *pmsFbo, GLuint *pmsColorTex, GLuint *pmsDepthStencilTex) {
   const int samples = 4;
 
-  int msWidth, msHeight;
-  windowsystem::GetScreenSize(&msWidth, &msHeight);
+  int msWidth=4096, msHeight=4096;
```

Comment:
Can we use the constant [here](https://github.com/exokitxr/exokit/pull/1036/files#diff-9072aaf78a49fe573d7263fd2ac67503R331) instead?

Reply:
changed in 7bff6b87